man/make.conf.5

```diff
@@ -533,6 +533,10 @@ If your GPG is auto unlocked on login, you do not need this.
 .B icecream
 Enable portage support for the icecream package.
 .TP
+.B ignore\-mtime
+Do not update mtime if the target file is equal. This can be useful on some
+filesystems to better utilize features like snapshots and data deduplication.
```

Comment:
We should probably note here that this feature violates the PMS stipulation about [preservation of file modification times](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-146001r1).

Reply:
Good point. I added a note with 29d4c2e